src/vivarium_public_health/testing/utils.py

```diff
@@ -30,9 +31,41 @@ def make_uniform_pop_data(age_bin_midpoint=False):
             "year_start": year_starts,
             "year_end": year_ends,
             "location": locations,
-            "value": [100] * len(mins),
+            "value": 100 * (np.array(maxes) - np.array(mins)),
         }
     )
     if age_bin_midpoint:  # used for population tests
         pop["age"] = pop.apply(lambda row: (row["age_start"] + row["age_end"]) / 2, axis=1)
     return pop
+
+
+def make_age_bins():
+    idx = pd.MultiIndex.from_tuples(
+        [
+            (0.0, 0.01917808, "Early Neonatal"),
```

Comment:
We actually ran into this yesterday. Is there a reason we do not include the "birth" age group in this?